### PR TITLE
ITG Minimaid patches

### PR DESCRIPTION
**Pull request by mooinglemur**
I wrote these based on an unfinished implementation by buzzert to get lights and native pad input from the Minimaid I/O board into openitg back in 2014.  These changes should merge cleanly.

### CODE REVIEW

**Comment by Matt1360** [comment]
:+1:

Clean and works.

File: src/arch/InputHandler/InputHandler_MiniMaid.cpp [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+#define DEVICE_INPUTDEVICE DEVICE_JOY1 // arbitrary
+#define INPUT_BIT_OFFSET 24
+#define INPUT_RANGE 23
+
+// Lights stuff
+static const uint64_t CABINET_LIGHTS = (1 * 8);
+static const uint64_t PAD_LIGHTS     = (2 * 8);
+static const uint64_t BASS_LIGHTS    = (4 * 8);
+
+static const uint64_t ALL_ON = 0x10F0FFC00;
```
Comment (spigwitmer) [inline src/arch/InputHandler/InputHandler_MiniMaid.cpp]: gcc6 likes this. unfortunately gcc3.3 (the one used for arcade builds) does not without the explicit unsigned long long suffix. Is this supposed to be `0x10F0FFC00ULL`?

Comment (Matt1360) [inline src/arch/InputHandler/InputHandler_MiniMaid.cpp]: Hey Pat (long time to see), it looks like `ALL_ON` isn't actually used anywhere.  It'd be nice to have it there for completeness, but it doesn't look like it's useful.

@mooinglemur you can probably either remove this, or change it to `0x10f0ffc00ULL` as infamouspat suggested.

**Comment by mooinglemur** [comment]
Agreed @Matt1360, since it's not used anywhere else in code, I converted it to a comment.

**Pull request merged**